security/wazuh-agent/README.md

```diff
@@ -0,0 +1,5 @@
+# OPNsense Wazuh Agent Plugin
+
+Plugin documentation: https://help-cloudfence.eu.helpdocs.com/opnsense-support/opnsense-wazuh-plugin
+
+**Cloudfence has sponsored the development of this plugin.**
```

Comment:
no advertisements please.

Reply:
I don't think it helps the project to scatter plugin documentation around the globe. Announcements in the forum are fine, when hosting documentation elsewhere, you might as well host the plugin somewhere else too. 

I'm not sure we made some sort of document about advertisements in the plugins, but we have been following the same rules internally for quite some time. It might also be a good idea to review the copyright notes from the files, some boilerplate seems to be missing the original authors. 

Adding the documentation later is no problem at all, we just don't want non functional links to external sources in the plugins. ( as example a link to https://documentation.wazuh.com/ is fine as it explains the origins of the software involved, there are quite some examples in our repository for these cases).

I haven't reviewed the plugin yet, if you plan to remove the advertisements, I'll put it on my list.

(p.s. contributing code for us isn't seen as sponsoring as we do need to spend time as well on it, which comes from our budget)